app/src/main/java/eu/kanade/presentation/manga/components/MangaToolbar.kt

```diff
@@ -47,118 +39,106 @@ fun MangaToolbar(
 
     // For action mode
     actionModeCounter: Int,
+    onCancelActionMode: () -> Unit,
     onSelectAll: () -> Unit,
     onInvertSelection: () -> Unit,
 
     modifier: Modifier = Modifier,
-    backgroundAlphaProvider: () -> Float = titleAlphaProvider,
+    backgroundAlphaProvider: () -> Float,
 ) {
-    Column(
+    val isActionMode by remember(actionModeCounter) {
+        derivedStateOf { actionModeCounter > 0 }
+    }
```

Comment:
```suggestion
    val isActionMode by remember(actionModeCounter) { actionModeCounter > 0 }
```

Reply:
that produces a error: Type 'TypeVariable(T)' has no method 'getValue(Nothing?, KProperty<*>)' and thus it cannot serve as a delegate

Reply:
replace the `by` with `=`

Reply:
that would force a recomposition everytimes `actionModeCounter` is changed. I push the change as per your request anw.

Reply:
```kt
val isActionMode by remember(actionModeCounter) {
    derivedStateOf { actionModeCounter > 0 }
}
```
is same as
```
val isActionMode = remember(actionModeCounter) { actionModeCounter > 0 }
```
it's only effective if it works as
```kt
val isActionMode by remember {
    derivedStateOf { actionModeCounter > 0 }
}
```
which it doesn't